apps/calendar/js/bridge.js

```diff
@@ -0,0 +1,118 @@
+define(function(require, exports) {
```

Comment:
This is awesome. I like the idea of moving the frontend onto a temporary implementation of the api bindings so that we can switch over super seamlessly